benchmark/distinct_user_id/rapidjson_sax.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#ifdef SIMDJSON_COMPETITION_RAPIDJSON
+
+#include "distinct_user_id.h"
+#include <string.h>
+namespace distinct_user_id {
+
+using namespace rapidjson;
+
+struct rapidjson_sax {
+    struct Handler {
+        std::vector<uint64_t>* result;
```

Comment:
Here you could more simply write ` std::vector<uint64_t>& result`. Then you could use the constructor:

```
Handler(std::vector<uint64_t> &r) : result(r) { }
```